Tools/wasm/wasm_assets.py

```diff
@@ -124,7 +139,9 @@ def create_stdlib_zip(
                 continue
             if entry in OMIT_ABSOLUTE:
                 continue
-            if entry.name.endswith(".py") or entry.is_dir():
+            if entry in SKIP_COMPILE:
+                pzf.write(entry)
```

Comment:
I don't think that this is necessary with rawfs testing.

Reply:
Fair point. I do think this still could be useful for browser tests via `from test import autotest`, but if you want I can add that in a different PR.

Reply:
This won't work anyway. The ``wasm_assets.py`` file excludes all test files and test helpers anyway. We are going to need an option to include all files. I also think that regression tests need access to ``.py`` files as well.